doc/howto/storage_volumes.md

```diff
@@ -52,6 +52,7 @@ The following restrictions apply:
 - Custom storage volumes of {ref}`content type <storage-content-types>` `block` or `iso` cannot be attached to containers, but only to virtual machines.
 - To avoid data corruption, storage volumes of {ref}`content type <storage-content-types>` `block` should never be attached to more than one virtual machine at a time.
 - Storage volumes of {ref}`content type <storage-content-types>` `iso` are always read-only, and can therefore be attached to more than one virtual machine at a time without corrupting data.
+- Filesystem storage volumes can't be attached to virtual machines while they're running.
```

Comment:
Sorry my suggestion of "Filesystem" here has fallen foul of our spell checker which wants "File system".

Reply:
Done!